lib/tictactoe/player.ex

```diff
@@ -1,11 +1,6 @@
 defmodule TicTacToe.Player do
-  alias TicTacToe.Board
-  alias TicTacToe.ConsoleIO
-
   defstruct type: nil,
             name: nil,
-            marker: nil
-
-  def select_cell(:ai, %Board{cells: cells}, markers), do: Board.first_free(cells, markers)
-  def select_cell(:human, _board, _markers), do: ConsoleIO.input()
```

Comment:
I was a bit vague in our discussion earlier, but I think we should still keep these functions defined in this module. Since we're already have pattern matching on the `move` functions inside the `Game` module, we can define `select_cell` functions with different arities. For example, `select_cell/0` could refer to the human player and `select_cell/x` could refer to the AI. For the AI's `select_move`, we can just pass the matched AI player in order to grab the `logic`. Does that make sense?